components/settings/SettingsProfileMetadata.vue

```diff
@@ -66,12 +66,14 @@ const chooseIcon = (i: number, text: string) => {
         </CommonDropdown>
         <input
           v-model="form.fieldsAttributes[i - 1].name"
-          type="text" placeholder="Label"
+          type="text" placeholder-text-secondary
+          :placeholder="$t('settings.profile.appearance.profile_metadata_label')"
           input-base
         >
         <input
           v-model="form.fieldsAttributes[i - 1].value"
-          type="text" placeholder="Content"
+          type="text" placeholder-text-secondary
+          :placeholder="$t('settings.profile.appearance.profile_metadata_value')"
```

Comment:
@userquin would you add a comment to this PR showing how you would like we announce these two new key?

Reply:
The message should be this:

New entries to translate (only included in `en-` and `pl-PL`):
- `settings.profile.appearance.profile_metadata_label`
- `settings.profile.appearance.profile_metadata_value`

Reply:
We only need to copy the source message and paste it on i18n channel.